lib/sycamore/sycamore/query/operators/query_database.py

```diff
@@ -33,7 +33,7 @@ class QueryDatabase(LogicalOperator):
                     },
                     {
                         "match": {
-                            "properties.path.keyword": "/path/to/data/*.pdf",
+                            "properties.path": "/path/to/data/*.pdf",
```

Comment:
I'm confused by this. We want is to use the `.keyword` version of the field so the wildcard match will work. I realize more few-shots may be needed to get this dialed in but I don't think you want to take `.keyword` out here.